app/styles/components/_panel.scss

```diff
@@ -86,6 +86,10 @@ $panel-content-max-width: 500px;
   background-color: $Theme-color-white;
 }
 
+.Panel--solidLightBlue {
```

Comment:
Any updates on this? I don't know if we should enforce it because it's already been used on the rest of the project, but your call @joaojusto